docs/source/api/safe/iod/iod_index.rst

```diff
@@ -0,0 +1,8 @@
+IOD Module
+==========
```

Comment:
IOD stands for "Initial Orbit Determination" (also known as "preliminary orbit determination"), which is a generic field that deals with computing an osculating orbit from a set of observations. There are other interesting algorithms for three observations (for example the Gauss method, and also https://github.com/aerospaceresearch/orbitdeterminator/). It would be nice to have a paragraph about that, and also explaining that `izzo` and `vallado` are just two algorithms to solve the Lambert problem.